keyboards/crkbd/keymaps/frogpad/keymap.c

```diff
@@ -0,0 +1,176 @@
+  //|--------+--------+--------+--------+--------+--------|                    |--------+--------+--------+--------+--------+--------|
+      KC_MINS, KC_HASH,  KC_DLR, KC_PERC,    KC_0, KC_COMM,                      KC_COMM,    KC_0, KC_PERC,  KC_DLR, KC_HASH, KC_MINS,\
+  //|--------+--------+--------+--------+--------+--------+--------|  |--------+--------+--------+--------+--------+--------+--------|
+                                          XXXXXXX, XXXXXXX,    OGUI,       OGUI, XXXXXXX, XXXXXXX \
+                                      //`--------------------------'  `--------------------------'
+    ),
+
+    [_EXT] = LAYOUT( \
+/*I recommend adding a TO() that leads to your two-handed layer to the first and last key in the next line of code if you 
+are adding this keymap to a two-handed keymap. It is left blank for this purpose.*/
+  //,-----------------------------------------------------.                    ,-----------------------------------------------------.
+      XXXXXXX, KC_WBAK, KC_HOME,   KC_UP,  KC_END, KC_WFWD,                      KC_WBAK, KC_HOME,   KC_UP,  KC_END, KC_WFWD, XXXXXXX,\
+  //|--------+--------+--------+--------+--------+--------|                    |--------+--------+--------+--------+--------+--------|
+      XXXXXXX,    ATAB, KC_LEFT, KC_DOWN, KC_RGHT,  KC_DEL,                       KC_DEL, KC_LEFT, KC_DOWN, KC_RGHT,    ATAB, XXXXXXX,\
+  //|--------+--------+--------+--------+--------+--------|                    |--------+--------+--------+--------+--------+--------|
+      XXXXXXX, C(KC_Z), C(KC_X), C(KC_C), C(KC_V),    OALT,                         OALT, C(KC_V), C(KC_C), C(KC_X), C(KC_Z), XXXXXXX,\
+  //|--------+--------+--------+--------+--------+--------+--------|  |--------+--------+--------+--------+--------+--------+--------|
+                                             OSFT,    OCTL,    CBSP,       CBSP,    OCTL,    OSFT \
+                                      //`--------------------------'  `--------------------------'
+    )
+  
+/* You should add these layers on to your existing keymap so you can toggle between one-handed usage and two-handed usage. Don't 
+forget to change the first keycode in _EXT to whatever you name your default layer aside from the frogpad layers. If you add 
+a TO(_FROG) somewhere on your default layer, you can easily switch between the two.
+  
+Make sure to add "#include "g/keymap_combo.h"" to the top of your keymap.c and add combos.def to the keyboard folder. 
+Also add "VPATH  +=  keyboards/gboards/" and "COMBO_ENABLE = yes" to rules.mk."
+  
+If you're having trouble with the OSM modifiers include the code for individual tapping terms in this keymap. 
+You must also use the last six 'define's in the config.h file. */
+
+};
+
+int RGB_current_mode;
+
+void persistent_default_layer_set(uint16_t default_layer) {
+  eeconfig_update_default_layer(default_layer);
+  default_layer_set(default_layer);
+}
```

Comment:
Could you remove this function?  

It's been deprecated in favor of the `set_single_persistent_default_layer` function (which is a core function).

https://docs.qmk.fm/#/ref_functions?id=setting-the-persistent-default-layer